packages/ui/react-ui-editor/src/components/TextEditor/TextEditor.tsx

```diff
@@ -254,7 +272,7 @@ export const defaultSlots: TextEditorSlots = {
     className: focusRing,
   },
   editor: {
-    className: 'min-bs-full',
+    className: 'h-full overflow-scroll',
```

Comment:
```suggestion
    className: 'bs-full overflow-scroll',
```

Reply:
It’s vitally important to never use `overflow-scroll`. We should find a way to remove it from the design system entirely.

Furthermore, here it is applied to an element that does not overflow (your approach maintains that `cm-scroller` has the scrollbar), so it forces it to apply padding for a scrollbar that it will never use. See screenshots:

<img width="985" alt="Screenshot 2024-02-26 at 09 54 15" src="https://github.com/dxos/dxos/assets/855039/6292208d-696a-4876-8851-779837ca2d07">
<img width="369" alt="Screenshot 2024-02-26 at 09 51 23" src="https://github.com/dxos/dxos/assets/855039/556d357a-6fc7-49c1-addb-83804f2dac3c">